### DISCUSSION TITLE
Public Functions Missing

### DISCUSSION THREAD

**Discussion by StingzLD** [sanderkl/PSMeraki#21] [General]
After installing and attempting to write scripts using this, I kept running into issues running some of the Public functions. Turns outs that the ones I was having issues with (listed below) are actually being hidden. I ended up uninstalling the module, then reinstalling it, but I had the same issue. A colleague who started using this module before me checked what commands are publicly available on his machine, as well, and these are also being hidden for him.

```
Get-MrkRestApiVersion
Invoke-MrkRestMethod
Restart-MrkDevice
Set-MrkNetworkSiteToSiteVPN
Set-MrkRestApiVersion
```
I then tried invoking the module, and sure enough, the commands are technically there, but not public. Looks like the PSD1 that gets installed is not the same one on GitHub. The one on GH uses wildcards for exporting the functions and commands, whereas the actual file that gets installed has specific ones listed. After adding those to the list, they are working no problem.

Could you please officially update this module?

Thank you,
Corey

**Comment by StingzLD** [comment]
Created Pull Request "Added Missing Public Functions"

**Comment by sanderkl** [comment]
Fixed it in 2.0.1, cheers
